sdks/python/apache_beam/runners/worker/bundle_processor.py

```diff
@@ -430,13 +424,13 @@ def clear(self):
 
   def _commit(self):
     if self._cleared:
-      self._state_handler.blocking_clear(self._state_key)
+      self._state_handler.clear(self._state_key, is_cached=True).get()
     if self._added_elements:
-      value_coder_impl = self._value_coder.get_impl()
-      out = coder_impl.create_OutputStream()
-      for element in self._added_elements:
-        value_coder_impl.encode_to_stream(element, out, True)
-      self._state_handler.blocking_append(self._state_key, out.get())
+      self._state_handler.append(
```

Comment:
An explicit comment regarding the need for blocking call might be useful here.

Reply:
Will consolidate with the above.